src/dataworker/DataworkerClientHelper.ts

```diff
@@ -102,7 +103,16 @@ export function getSpokePoolClientEventSearchConfigsForFastDataworker(
   config: DataworkerConfig,
   clients: DataworkerClients,
   dataworker: Dataworker
-) {
+): {
+  fromBundle: ProposedRootBundle;
+  toBundle: ProposedRootBundle;
+  fromBlocks: {
+    [k: string]: number;
+  };
+  toBlocks: {
+    [k: string]: number;
+  };
```

Comment:
These are pretty verbose; worthwhile condensing them?

```suggestion
  fromBlocks: { [k: string]: number };
  toBlocks: { [k: string]: number };
```